scripts/globals/weaponskills.lua

```diff
@@ -1043,6 +1048,11 @@ function handleWSGorgetBelt(attacker)
                 break;
             end
         end
+
+        if (belt == 28420) then
```

Comment:
not a fan of using ids but guess there isnt really a better way
a comment with which item it is would be nice though, for anyone who needs to look up how this is handled later